source/frontend/src/models/api/Lease.ts

```diff
@@ -17,6 +17,7 @@ export interface Api_Lease {
   initiatorType?: Api_TypeCode<string>;
   type?: Api_TypeCode<string>;
   statusType?: Api_TypeCode<string>;
+  accountType?: Api_TypeCode<string>;
```

Comment:
hmm, does this get used anywhere? I didn't think anything got added to the db or backend models for this story.